.gitignore

```diff
@@ -41,3 +41,5 @@ yarn-debug.log*
 .yarn-integrity
 
 config/database.yml
+
+.tool-versions
```

Comment:
remove `.tool-versions`. we need this one to help other contributors for setting up this repository

Reply:
@Weltinio @rjmssevilla